autorest/azure/environments.go

```diff
@@ -0,0 +1,99 @@
+package azure
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+)
+
+// Environment represents a set of endpoints for each of Azure's Clouds.
+type Environment struct {
+	Name                      string
+	ManagementPortalURL       string
```

Comment:
I'd prefer to leave it. The .NET SDK specifies it. I can maybe imagine a scenario of wanting to link to a template vm deployment and giving the Portal URI for it. At the very least, I don't think it hurts anyone :)